sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -270,25 +255,23 @@ def prepare_data(
     #     "--instrument", "mag",
     #     "--data-level", "l1a",
     #     "--start-date", "20231212",
-    #     "--end-date", "20231212",
     #     "--version", "v00-01",
     #     "--dependency", """[
     #         {
     #             'instrument': 'swe',
     #             'data_level': 'l0',
     #             'descriptor': 'lveng-hk',
     #             'start_date': '20231212',
-    #             'end_date': '20231212',
     #             'version': 'v01-00',
```

Comment:
Perhaps outside the scope of this PR, but the version format examples here now need to be updated.

Reply:
Yes. Good catch! Will update comments.